sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusTestEnvironment.cs

```diff
@@ -95,21 +83,55 @@ public ServiceBusTestEnvironment() : base("servicebus")
         ///
         public string SharedAccessKey => ParsedConnectionString.Value.SharedAccessKey;
 
+        /// <summary>
+        ///   The Azure Authority host to be used for authentication with the active cloud environment.
+        /// </summary>
+        ///
+        public new string AuthorityHostUrl => base.AuthorityHostUrl ?? "https://login.microsoftonline.com/";
```

Comment:
To be transparent, I'm not in love with this, but I'm explicitly shadowing to allow for assuming defaults without having to force that behavior on the base class. I found it confusing to have unique members with similar names to the base and wanted to ensure that the defaults were encapsulated here rather than at each call site.